python_modules/dagster/dagster/_core/definitions/asset_reconciliation_sensor.py

```diff
@@ -650,13 +650,16 @@ def determine_asset_partitions_to_reconcile_for_freshness(
             current_data_times = get_current_data_times_for_key(
                 instance_queryer, asset_graph, relevant_upstream_keys, key
             )
+            expected_data_times = get_expected_data_times_for_key(
```

Comment:
I don't think this line is necessary (if it needs to be there for linting reasons, we should set it to the empty dictionary here)